installscripts/jazz-terraform-unix-noinstances/scripts/scmpush.sh

```diff
@@ -25,7 +25,12 @@ if [ ! -d ./jazz-core-scm ] ; then
 fi
 
 cd ./jazz-core
-find . -name ".git*" -exec rm -rf '{}' \;  -print
+
+# Condition to include "gitlab-build-pack" for gitlab Scenario3 to SCM repos.
+if [ $scm == "gitlab" ]; then
+  find . -name ".git*" -exec rm -rf '{}' \;  -print
```

Comment:
Can we use a more specific matching pattern? Tomorrow we will migrate to even more SCM's, we want to make sure we can include them based on a "known" matching pattern for a given SCM. Instead of naming pattern, we can even have specific file to be included if scm == x or scm == y.